system-addon/bootstrap.js

```diff
@@ -124,7 +124,7 @@ function onBrowserReady() {
   migratePref("browser.newtabpage.rows", rows => {
     // Just disable top sites if rows are not desired
     if (rows <= 0) {
-      Services.prefs.setBoolPref("browser.newtabpage.activity-stream.showTopSites", false);
+      Services.prefs.setBoolPref("browser.newtabpage.activity-stream.feeds.topsites", false);
```

Comment:
We'll want to migrate `showTopSites` pref too. Otherwise anyone who turned off top sites via the UI would get it back.